lib/services/components.js

```diff
@@ -446,7 +446,7 @@ function getPossibleTemplates(name, templateName) {
 /**
  * Get the first template found for a particular component
  * @param {string} name  i.e., 'article', 'paragraph', 'content', 'my-layout'
- * @param {string} templateName  i.e., 'template', 'sitemap', 'apple', 'facebook'
+ * @param {string} templateName  i.e., 'template', 'apple', 'facebook'
```

Comment:
The one on line 433 has brackets around `templateName` but this one doesn't, what's the difference?